apps/testclient/templates/authorize.html

```diff
@@ -37,6 +37,9 @@ <h3 class="ds-c-alert__heading">You'll need sample beneficiary credentials to lo
 			<div class="action-container">
 				<a href="{{authorization_url}}" class="ds-c-button ds-c-button--solid">Authorize as a Beneficiary</a>
 			</div>
+			<div class="action-container">
+				<a href="{{authorization_url}}&lang=es" class="ds-c-button ds-c-button--solid">Authorize as a Beneficiary (Spanish)</a>
+			</div>
```

Comment:
`action-container` gives the box a rounded bottom border, so we should put both buttons into the same `action-container`. One potential fix could be this, though you may want to add a line break or something to get the spacing looking okay:
```suggestion
			<div class="action-container">
				<a href="{{authorization_url}}" class="ds-c-button ds-c-button--solid">Authorize as a Beneficiary</a>
				<a href="{{authorization_url}}&lang=es" class="ds-c-button ds-c-button--solid">Authorize as a Beneficiary (Spanish)</a>
			</div>
```
Also since this involves changes to the test client, we should make sure we get product/design review on this as well before delivering.

Reply:
When I click the spanish link and then go back and click the english link in the test client, it is still in Spanish. That might be a browser-specific issue, but maybe we should add the english parameter to the url for the English one.

Reply:
> When I click the spanish link and then go back and click the english link in the test client, it is still in Spanish. That might be a browser-specific issue, but maybe we should add the english parameter to the url for the English one.

cache behavior, will explicitly add a lang=en, see if that fix the issue